posthog/clickhouse/migrations/0111_property_definitions_table.py

```diff
@@ -0,0 +1,35 @@
+from posthog.clickhouse.client.migration_tools import run_sql_with_exceptions
+from posthog.clickhouse.client.connection import NodeRole
+from posthog.settings.data_stores import POSTHOG_MIGRATION_CLUSTER
+
+PROPERTY_DEFINITIONS_TABLE_SQL = f"""
+CREATE TABLE property_definitions ON CLUSTER '{POSTHOG_MIGRATION_CLUSTER}'
```

Comment:
`run_sql_with_exceptions` will execute the creation on all nodes specified by NodeRole, no need to add the on cluster part here!

```suggestion
CREATE TABLE property_definitions
```